00.prerequisites/00.ad.documentation.md

```diff
@@ -0,0 +1,105 @@
+Checking which version of Python is installed:
```

Comment:
I do not see benefits in adding another file to this directory.  If you would like to improve something - please state the problem and add information under releavant file name or into README